graylog2-web-interface/src/views/components/ViewActionsMenu.jsx

```diff
@@ -106,7 +106,7 @@ ViewActionsMenu.propTypes = {
   metadata: PropTypes.shape({
     undeclared: ImmutablePropTypes.Set,
   }).isRequired,
-  view: PropTypes.instanceOf(View).isRequired,
+  view: PropTypes.object.isRequired,
```

Comment:
We talked about the unreliable `instanceof` check in the past, it would be great to find a reliable solution, but for now I've just typed the prop as an object, like we do it in many other places.